# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../escreen/DESTDIR.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- escreen-0.0.3/build/bottom.make.vanilla	2007-03-08 15:19:29.012476022 +0000
+++ escreen-0.0.3/build/bottom.make	2007-03-08 15:20:04.554501446 +0000
@@ -30,9 +30,9 @@
   install:: $($(X_MODULE)_BINARY)
 	$(Q)for x in $^; do \
 	  case $$x in \
-		*$(X_DYNEXT) ) echo INSTALL DYNLIB $$x; install $$x $(libdir) ;; \
+		*$(X_DYNEXT) ) echo INSTALL DYNLIB $$x; install $$x $(DESTDIR)$(libdir) ;; \
 		*$(X_LIBEXT) ) ;; \
-		*$(X_EXEEXT) ) echo INSTALL EXEC   $$x; install $$x $(bindir) ;; \
+		*$(X_EXEEXT) ) echo INSTALL EXEC   $$x; install $$x $(DESTDIR)$(bindir) ;; \
 	  esac ;\
 	done
 endif
